src/components/layout/Layout.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { ReactNode } from 'react'
+import MainNavigation from './MainNavigation'
+import Footer from './Footer'
+import Header from './Header'
+
+type Props = {
+    children?: ReactNode
+}
+
+const Layout: React.FC<Props> = (props) => {
```

Comment:
destructure the props, we do not use "props"

Reply:
({ children }: Props
)

Reply:
ok done